src/apps/common/apputils.c

```diff
@@ -699,12 +711,14 @@ int handle_socket_error() {
      * Must close connection.
      */
     return 0;
+#if defined(__linux__)
```

Comment:
You might be better off with a not MSVC flag, as `EHOSTDOWN` is BSD and others also